src/AccessibilityInsights.SharedUx/Controls/ScannerResultControl.xaml.cs

```diff
@@ -123,9 +123,9 @@ private void SetScannerResultTreeView(A11yElement e)
         private void UpdateTree()
         {
             IEnumerable<ScanListViewItemViewModel> itemViewModel = from l in this.List
```

Comment:
This goes really far to the right--I might introduce a local variable just to move everything to the left

Reply:
This was already a local variable. I changed the type to `var` to reduce the indent a little bit